src/components/utils/test/date_time_test.cc

```diff
@@ -108,11 +108,10 @@ TEST(DateTimeTest, CalculateTimeSpan) {
   // arrange
   const TimevalStruct time = date_time::DateTime::getCurrentTime();
 
-  const uint32_t sleep_time_mSec = 10;
+  const uint64_t sleep_time_mSec = 100;
   threads::sleep(sleep_time_mSec);
-
   // assert
-  ASSERT_GE(date_time::DateTime::calculateTimeSpan(time), sleep_time_mSec);
+  ASSERT_GE(date_time::DateTime::calculateTimeSpan(time) + 1, sleep_time_mSec);
```

Comment:
@orhan-mehmedov @OHerasym 
Looks like we need just `ASSERT_GT` isntead of `ASSERT_GE`.